code/scripting/api/libs/base.cpp

```diff
@@ -214,6 +214,21 @@ ADE_FUNC(createRandomVector, l_Base, nullptr, "Creates a fairly random normalize
 	return ade_set_args(L, "o", l_Vector.Set(v3));
 }
 
+ADE_FUNC(createRandomOrientation, l_Base, nullptr, "Creates a fairly random orientation object.", "orientation", "Orientation object")
```

Comment:
I don't think we have to hedge our bets here with "fairly", I am quite sure this is uniformly random in the space of all possible matrices.

Reply:
Okay